opensearch/services/publications/search.php

```diff
@@ -52,6 +52,16 @@
     $constraints['years'] = explode(";",$_GET['years']);
 }
 
+// Determine if we retrieve only DCO publications
+if (@$_GET['dcoPubs'] && @$_GET['dcoPubs'] != '') {
+	$constraints["dcoPubs"] = explode(";",$_GET['dcoPubs']);
+}
+else
+{
+    $constraints["dcoPubs"] = array() ;
+    $constraints["dcoPubs"][0] = "yes" ;
```

Comment:
When the first set of requests come in dcoPubs isn't set yet, like there's no selection. So yes, I default it to yes.